network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -109,7 +109,11 @@ class Meta:
 
 
 class BlogPage(FoundationMetadataPageMixin, Page):
-    body = StreamField(base_fields)
+    body = StreamField(
+        base_fields + [
+            ('airtable', customblocks.AirTableBlock()),
```

Comment:
what other pages use `base_fields`? Because depending on those, it might make sense to just add this to the full list.